src/20240528_Multi_GHOCrossChainLaunch/GHOCrossChainLaunch_20240528.s.sol

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {GovV3Helpers, IPayloadsControllerCore, PayloadsControllerUtils} from 'aave-helpers/GovV3Helpers.sol';
+import {GovernanceV3Ethereum} from 'aave-address-book/GovernanceV3Ethereum.sol';
+import {EthereumScript, ArbitrumScript} from 'aave-helpers/ScriptUtils.sol';
+import {AaveV3Ethereum_GHOCrossChainLaunch_20240528} from './AaveV3Ethereum_GHOCrossChainLaunch_20240528.sol';
+import {AaveV3Arbitrum_GHOCrossChainLaunch_20240528} from './AaveV3Arbitrum_GHOCrossChainLaunch_20240528.sol';
+import {AaveV3Arbitrum_GHOCrossChainListing_20240528} from './AaveV3Arbitrum_GHOCrossChainListing_20240528.sol';
+
+/**
+ * @dev Deploy Ethereum
+ * deploy-command: make deploy-ledger contract=src/20240528_Multi_GHOCrossChainLaunch/GHOCrossChainLaunch_20240528.s.sol:DeployEthereum chain=mainnet
+ * verify-command: FOUNDRY_PROFILE=mainnet npx catapulta-verify -b broadcast/GHOCrossChainLaunch_20240528.s.sol/1/run-latest.json
+ */
+contract DeployEthereum is EthereumScript {
+  function run() external broadcast {
+    // deploy payloads
+    address payload0 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Ethereum_GHOCrossChainLaunch_20240528).creationCode
+    );
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](1);
+    actions[0] = GovV3Helpers.buildAction(payload0);
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
+  }
+}
+
+/**
+ * @dev Deploy Arbitrum
+ * deploy-command: make deploy-ledger contract=src/20240528_Multi_GHOCrossChainLaunch/GHOCrossChainLaunch_20240528.s.sol:DeployArbitrum chain=arbitrum
+ * verify-command: FOUNDRY_PROFILE=arbitrum npx catapulta-verify -b broadcast/GHOCrossChainLaunch_20240528.s.sol/42161/run-latest.json
+ */
+contract DeployArbitrum is ArbitrumScript {
+  function run() external broadcast {
+    // deploy payloads
+    address payload0 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Arbitrum_GHOCrossChainLaunch_20240528).creationCode
+    );
+    address payload1 = GovV3Helpers.deployDeterministic(
+      type(AaveV3Arbitrum_GHOCrossChainListing_20240528).creationCode
+    );
+
+    // compose action
+    IPayloadsControllerCore.ExecutionAction[]
+      memory actions = new IPayloadsControllerCore.ExecutionAction[](2);
+    actions[0] = GovV3Helpers.buildAction(payload0);
+    actions[1] = GovV3Helpers.buildAction(payload1);
+
+    // register action at payloadsController
+    GovV3Helpers.createPayload(actions);
```

Comment:
This way there will be two actions and a single payload, so payload 0 and payload 1 would be executed in a single tx which is not what we wish. 
For payload0 and payload1 lets deploy separate payloads, for reference you can check [here](https://github.com/bgd-labs/aave-proposals-v3/blob/main/src/20240422_Multi_MigrateRobotsToChainlinkAutomationV2/MigrateRobotsToChainlinkAutomationV2_20240422.s.sol#L43-L52).
Also proposal creation script should be updated accordingly as well.